lagom14-scala-service-locator-dns/src/test/scala/com/lightbend/lagom/scaladsl/dns/DnsServiceLocatorSpec.scala

```diff
@@ -54,9 +55,6 @@ class DnsServiceLocatorSpec extends WordSpec with Matchers with BeforeAndAfterAl
     override lazy val materializer: Materializer = mat
     override lazy val executionContext: ExecutionContext = actorSystem.dispatcher
     override lazy val router: Router = Router.empty
-    override lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
-    override lazy val circuitBreakerConfig: CircuitBreakerConfig = new CircuitBreakerConfig(configuration)
-    override lazy val circuitBreakers: CircuitBreakers = new CircuitBreakers(actorSystem, circuitBreakerConfig, circuitBreakerMetricsProvider)
```

Comment:
I don't know why we have it in test. It's totally not needed as this is exactly what the `CircuitBreakersComponent` does.